apis/catalog/v1alpha1/types.go

```diff
@@ -30,3 +30,21 @@ type UpdateConstraints struct {
 	// An empty list indicates no version is rejected.
 	Denylist []string `json:"denylist,omitempty"`
 }
+
+type ArchiverSpec struct {
+	Walg           WalgSpec           `json:"walg"`
+	CSISnapshotter CSISnapshotterSpec `json:"csiSnapshotter"`
```

Comment:
CSISnapshotter is just a driver we are using for full-db backup. so, This field-name should be more generic, for example, 
FullDatabase FullDatabaseArchiverSpec
````
type struct FullDatabaseArchiverSpec {
    CSISnapshotterAddonName string
}
````

Reply:
Yes, but it's for db versions crd. So, there will be different struct for different drivers.

Reply:
I would prefer this style, 
```
FullDB FullDBArchiverSpec

type struct FullDBArchiverSpec {
    CSISnapshotter   CSISnapshotterSpec
    // other drivers here
}
type CSISnapshotterSpec struct {
    AddonName string
}
```